sandbox/apparmor/notify/listener/export_test.go

```diff
@@ -55,6 +55,10 @@ func FakeRequestWithIDVersionClassAllowDeny(id uint64, version notify.ProtocolVe
 	}
 }
 
+func MockUnixGetpgid(f func(pid int) (pgid int, err error)) (restore func()) {
+	return testutil.Mock(&unixGetpgid, f)
+}
+
 func MockOsOpen(f func(name string) (*os.File, error)) (restore func()) {
 	restore = testutil.Backup(&osOpen)
```

Comment:
maybe a little drive-by tweak while you're already modifying the surrounding code:
```
return testutil.Mock(&osOpen, f)
```